src/bot.ts

```diff
@@ -435,6 +432,14 @@ export class DiscordBot {
         return dbEmoji.MxcUrl;
     }
 
+    public async GetEmojiByMxc(mxc: string): Promise<DbEmoji> {
```

Comment:
I wonder if this should have gone on `Store`

Reply:
Or better yet just implemented as a private function for whoever needed it.

Reply:
Soru put it there as the normal `GetEmoji` method was already there, so if one is moved, both should be - should they be moved? Maybe this would belong better into a different PR?

Reply:
Eh, leave it for now and we can come back to it when we can.